tls/s2n_client_hello.c

```diff
@@ -144,7 +146,7 @@ int s2n_collect_client_hello(struct s2n_connection *conn, struct s2n_stuffer *so
     return 0;
 }
 
-static int s2n_parse_client_hello(struct s2n_connection *conn)
```

Comment:
I would recommend against making a static function a non-static. Was the goal to unit test this function? Maybe use `s2n_client_hello_recv` instead?

Reply:
Yeah, I could use that function. I also could just add #include "s2n_client_hello.c" to the test file. I wasn't sure if s2n has a standard way of testing static functions.